server/models/seed_models.go

```diff
@@ -0,0 +1,65 @@
+package models
+
+import (
+
+	"os"
+	"path/filepath"
+	"github.com/layer5io/meshkit/logger"
+	meshmodel "github.com/layer5io/meshkit/models/meshmodel/registry"
+	"github.com/layer5io/meshkit/models/registration"
+)
+
+var ModelsPath = "../meshmodel"
+
+func getModelDirectoryPaths() ([]string, error) {
```

Comment:
@humblenginr I think this function can be now safely removed, as the implementation of `PkgUnit` for `Directory` handles it already.